### PR TITLE
Fix TransitioningContentControl Loaded/Unloaded

### PR DESCRIPTION
**Pull request by kronic**
Fix https://github.com/reactiveui/ReactiveUI/issues/2422

### CODE REVIEW

File: src/ReactiveUI.Wpf/Themes/Generic.xaml [hunk 1]
```diff
@@ -458,7 +458,7 @@
                                     <Storyboard>
                                         <DoubleAnimation 
                                             BeginTime="00:00:00" Duration="00:00:00.2"
-                                            Storyboard.TargetName="PART_PreviousContentPresentationSite"
```
Comment (glennawatson) [inline src/ReactiveUI.Wpf/Themes/Generic.xaml:461]: There are frameworks out there using these storyboard names at the moment so probably best you don't rename them for the moment.

File: src/ReactiveUI.Wpf/TransitioningContentControl.cs [hunk 2]
```diff
@@ -151,7 +152,7 @@ public enum TransitionDirection
         /// Gets or sets the transition duration.
         /// </summary>
         /// <value>The duration.</value>
-        public TimeSpan Duration { get => (TimeSpan)GetValue(TransitionDurationProperty); set => SetValue(TransitionDurationProperty, value); }
+        public TimeSpan Duration { get => (TimeSpan)GetValue(DurationProperty); set => SetValue(DurationProperty, value); }
```
Comment (glennawatson) [inline src/ReactiveUI.Wpf/TransitioningContentControl.cs:155]: This is a breaking change and not keen changing it.

**Review by glennawatson — CHANGES_REQUESTED** [review]

File: src/ReactiveUI.Wpf/Themes/Generic.xaml (hunk 1 above)
Comment (kronic) [reply src/ReactiveUI.Wpf/Themes/Generic.xaml:461]: @glennawatson Type changes anyway ContentPresenter -> Image

File: src/ReactiveUI.Wpf/TransitioningContentControl.cs (hunk 2 above)
Comment (kronic) [reply src/ReactiveUI.Wpf/TransitioningContentControl.cs:155]: @glennawatson ok

**Comment by glennawatson** [comment]
It will be a couple days before I will be able to merge this in. Want to test the changes and probably benchmark as well to compare. 

Thanks for submitting it.

**Comment by kronic** [comment]
@glennawatson There are test results?

**Comment by glennawatson** [comment]
I'll merge it in and tweak as needed. Thanks. Most likely will be changing the animation names back to help external libraries out there that rely on them.

**Review by glennawatson — APPROVED** [review]

**Pull request merged**